cob_scan_unifier/src/scan_unifier_node.cpp

```diff
@@ -230,25 +233,25 @@ void scan_unifier_node::topicCallbackLaserScan(const sensor_msgs::LaserScan::Con
 sensor_msgs::LaserScan scan_unifier_node::unifieLaserScans()
 {
   sensor_msgs::LaserScan unified_scan = sensor_msgs::LaserScan();
-  std::vector<sensor_msgs::PointCloud> vec_cloud;
-  vec_cloud.assign(config_.number_input_scans, sensor_msgs::PointCloud());
+  std::vector<sensor_msgs::PointCloud2> temp_cloud;
+  temp_cloud.assign(config_.number_input_scans, sensor_msgs::PointCloud2());
+  std::vector<Cloud> vec_cloud;
+  vec_cloud.assign(config_.number_input_scans, Cloud());
 
   if(!vec_laser_struct_.empty())
   {
     ROS_DEBUG("start converting");
     for(int i=0; i < config_.number_input_scans; i++)
     {
-      vec_cloud.at(i).header.stamp = vec_laser_struct_.at(i).current_scan_msg.header.stamp;
+      temp_cloud.at(i).header.stamp = vec_laser_struct_.at(i).current_scan_msg.header.stamp;
       ROS_DEBUG_STREAM("Converting scans to point clouds at index: " << i << ", at time: " << vec_laser_struct_.at(i).current_scan_msg.header.stamp << " now: " << ros::Time::now());
       try
       {
-        listener_.waitForTransform("/base_link", vec_laser_struct_.at(i).current_scan_msg.header.frame_id,
-            vec_laser_struct_.at(i).current_scan_msg.header.stamp, ros::Duration(3.0));
-
```

Comment:
why do you remove this? (Could discuss, whether this makes sense, but is there any particular reason?